src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/SwapEndpointSpec.groovy

```diff
@@ -991,13 +994,19 @@ switches"() {
         def exc = thrown(HttpServerErrorException)
         new FlowEndpointsNotSwappedExpectedError(~/Not enough bandwidth or no path found/).matches(exc)
 
+        when: "Get actual sources of flow1 and flow2"
+        def actualFlow1Src = northbound.getFlow(flow1.id).source
+        def actualFlow2Src = northbound.getFlow(flow2.id).source
+
+        then: "Actual flow1, flow2 sources are different"
+        Set actualFlow1Set = [actualFlow1Src.datapath, actualFlow1Src.portNumber,
+                              actualFlow1Src.vlanId, actualFlow1Src.innerVlanId].toSet()
+        Set actualFlow2Set = [actualFlow2Src.datapath, actualFlow2Src.portNumber,
+                              actualFlow2Src.vlanId, actualFlow2Src.innerVlanId].toSet()
+        assert !(actualFlow1Set == actualFlow2Set)
```

Comment:
you can simplify the check:
`assert actualFlow1Set != actualFlow2Set`
Do we really need to collect actualFlow1Set/ actualFlow2Set data separately? What is the purpose of this collecting and following check?

Reply:
Thank you! Changed to `assert actualFlow1Set != actualFlow2Set`.
I am collecting the actual data of the flows to verify that after the failed attempt to swap the endpoints, the flow sources are not the same due to the bug #5635. Currently, each time the flows have the same source due to the effect which is not expected.